app/javascript/components/pages/Jobs.jsx

```diff
@@ -1,25 +1,37 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import propTypes from 'prop-types';
+import { useCookies } from 'react-cookie';
 import SharedLayout from 'components/layout/SharedLayout';
 import PageTitle from 'components/PageTitle';
 import Button from 'components/Button';
 import { getJobs } from '../../datasources';
 import { postedAtString } from '../../utils';
+import { UnauthorizedError } from '../../errors';
 
 import 'stylesheets/page';
 import 'stylesheets/jobs';
 
 const Jobs = () => {
     const [jobs, setJobs] = useState([]);
+    const [cookies] = useCookies();
 
     useEffect(() => {
         const fetchData = async () => {
-            const data = await getJobs();
-            setJobs(data);
+            try {
+                const data = await getJobs(cookies['wnb_job_board_token']);
+                setJobs(data);
+            } catch (error) {
+                if (error instanceof UnauthorizedError) {
```

Comment:
@stephanieminn Here's how I addressed your comment. This is a super Ruby-ish way so if you think there's a more JavaScript-y way to do it please let me know:
- I created a custom error class called `UnauthorizedError` that I raise only if the API returns 401 responses
- When catching an error, I can check for that class and add custom handling

Later, I can add error boundaries so that I can re-raise all other errors and then have the app communicate gracefully to the user that something went wrong.
-

Reply:
that sounds like a great solution to me!